src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,20 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static final int COUNT_FIGURES = 6;
 
+    public static void main(String[] args){
+        FigureSupplier supplier = new FigureSupplier();
+        Figure[] figures = new Figure[COUNT_FIGURES];
+        int i = 0;
+        for(; i < COUNT_FIGURES/2; i++){
```

Comment:
The loop condition `i < COUNT_FIGURES/2` assumes that `COUNT_FIGURES` is even. If there's a possibility of `COUNT_FIGURES` being odd, consider using integer division to ensure the correct number of iterations.